client/src/pages/Login.js

```diff
@@ -1,187 +1,74 @@
-                    <Paper className={style.card} />
-                    <Paper className={style.card} />
-                    <Paper className={style.card} />
-                    <Paper className={style.card} />
-                    <Paper className={style.card} />
+            <div className={classes.container}>
+                <div className={classes.grid}>
+                    <Paper className={classes.card} />
+                    <Paper className={classes.card} />
+                    <Paper className={classes.card} />
+                    <Paper className={classes.card} />
+                    <Paper className={classes.card} />
+                    <Paper className={classes.card} />
+                    <Paper className={classes.card} />
+                    <Paper className={classes.card} />
                 </div>
             </div>
-            <div className={style.signin}>
-                <div className={style.modal}>
-                    <h1 className={style.welcome}>Welcome!</h1>
-                    <div className={style.emailContainer}>
-                        <TextField
-                            label='Username'
-                            name='username'
-                            className={style.inputemail}
-                            onChange={e => onChangeText(e)}
-                        />
+            <div className={classes.signIn}>
+                <div className={classes.modal}>
+                    <h1 className={classes.welcome}>Welcome!</h1>
+                    <div className={classes.emailContainer}>
+                        <TextField label='E-mail' className={classes.inputEmail} onChange={handleChangeEmail}/>
                     </div>
-                    <div className={style.passContainer}>
-                        <TextField
-                            label='Password'
-                            type='password'
-                            name='password'
-                            className={style.inputpass}
-                            onChange={e => onChangeText(e)}
-                        />
+                    <div className={classes.passwordContainer}>
+                        <TextField label='Password' type="password" className={classes.inputPassword} onChange={handleChangePassword}/>
                     </div>
-                    <p className={style.forgotpass}>Forgot your password?</p>
-                    <button className={style.loginbutton} onClick={postInfo}>
-                        Login
-                    </button>
-                    <p className={style.footer}>
-                        Don't have an account?{' '}
-                        <Link to='/signup' className={style.signup}>
-                            Sign Up!
-                        </Link>
-                    </p>
+                    <p className={classes.forgotPassword}>Forgot your password?</p>
+                    <button className={classes.loginButton} onClick={postInfo}>Login</button>
```

Comment:
I noticed this in your project - is there any reason you are not using the material ui Button class or the Typography classes?

Reply:
no reason, I'm just more familiar with classic css and html